docs/deploying-kas-fleet-manager-to-openshift.md

```diff
@@ -198,7 +202,11 @@ make deploy/service IMAGE_TAG=<your-image-tag-here> <OPTIONAL_PARAMETERS>
 - `ADMIN_API_SSO_BASE_URL`: Base URL of admin API endpints SSO. Defaults to `"https://auth.redhat.com"`
 - `ADMIN_API_SSO_ENDPOINT_URI`: admin API SSO endpoint URI. defaults to `"/auth/realms/EmployeeIDP"`
 - `ADMIN_API_SSO_REALM`: admin API SSO realm. Defaults to `"EmployeeIDP"`
-
+- `KAFKA_TLS_CERTIFICATE_MANAGEMENT_MUST_STAPLE`: The tls certificate management must staple. Adds the must staple TLS extension to the certificate signing request. The default value is `false`
+- `KAFKA_TLS_CERTIFICATE_MANAGEMENT_STRATEGY`: The tls certificate management strategy. Possible options are manual and automaitic. The default value is `manual`
```

Comment:
What is manual and what is automatic?

Reply:
I can a phrase or two for each mode but the names should be descriptive enough :-)